spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/controller/ReleaseController.java

```diff
@@ -141,10 +141,19 @@ public ReleaseControllerLinksResource resourceLinks() {
 		return this.releaseResourceAssembler.toResource(release);
 	}
 
-	@RequestMapping(path = "/delete/{name}", method = RequestMethod.POST)
-	@ResponseStatus(HttpStatus.CREATED)
+	@RequestMapping(path = "/{name}", method = RequestMethod.DELETE)
+	@ResponseStatus(HttpStatus.OK)
+	public Resource<Release> delete(@PathVariable("name") String releaseName) {
+		Release release = this.skipperStateMachineService.deleteRelease(releaseName, new DeleteProperties());
```

Comment:
perhaps you should call the `ReleaseController#delete(releaseName, false)` instead of duplicating the implementation? Or perhaps factor out the implementation in private method and call it from both controller delete methods